protocol/app/app.go

```diff
@@ -1287,6 +1301,50 @@ func New(
 	app.MountTransientStores(tkeys)
 	app.MountMemoryStores(memKeys)
 
+	// Slinky setup
```

Comment:
We shld ensure that full-nodes aren't running the oracle? Is there a config parameter to control this?